src/plugins/maven-workspace.ts

```diff
@@ -310,6 +315,16 @@ export class MavenWorkspace extends WorkspacePlugin<MavenArtifact> {
     return graph;
   }
 
+  /**
+   * Given a release version, determine if we should bump the manifest
+   * version as well. For maven artifacts, SNAPSHOT versions are not
+   * considered releases.
+   * @param {Version} version The release version
+   */
+  protected isReleaseVersion(version: Version): boolean {
+    return !version.preRelease?.includes('SNAPSHOT');
```

Comment:
nit: might be worth pulling `SNAPSHOT` into a constant, if we use this value in a variety of places.